Project.toml

```diff
@@ -37,7 +37,7 @@ TensorCast = "02d47bb6-7ce6-556a-be16-bb1710789e2b"
 
 [compat]
 BioAlignments = "2, 3"
-BioStructures = "4"
+BioStructures = "4.1"
```

Comment:
This is the only change I see that narrows compat - was that intentional / why is it needed?

I think no, but narrowing compat isn't considered breaking for semver is it?

Reply:
https://github.com/BioJulia/BioStructures.jl/releases/tag/v4.1.0

We need the `format` keyword.

And no, it's not breaking to require new features, which is what the minor release number encodes. It's only breaking if you stop supporting previously-supported features.

I also think that breaking changes in dependencies is a gray area for semver.